features/sidebar-reset.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Manage WordPress sidebars
+
+  Scenario: Reset sidebar
+    Given a WP install
+
+    When I run `wp theme install p2 --activate`
+    Then STDOUT should not be empty
+
+    When I run `wp widget list sidebar-1 --format=count`
+    Then STDOUT should be:
+    """
+    6
+    """
```

Comment:
The indentation is slightly off here. It should be:

```
Then STDOUT should be:
  """
  6
  """
```

This comment applies to all instances throughout.